edb/edgeql/pygments/meta.py

```diff
@@ -1,7 +1,7 @@
 # AUTOGENERATED BY EdgeDB WITH
 #     $ edb gen-pygments-grammars edgeql
 #
-# TIMESTAMP: 2019-03-12 19:46:27
+# TIMESTAMP: 2019-03-14 13:13:13
```

Comment:
No need for this change. In fact, I think this was a bad idea of mine to include the TIMESTAMP at all.  We can always use git to see when the file was last updated.  So please drop the TIMESTAMP comment generation from `edg gen-meta-grammars`

Reply:
Done